apps/angular/src/app/features/anime/components/anime-table/anime-table.component.ts

```diff
@@ -38,4 +213,86 @@ export class AnimeTableComponent {
   public trackBy(_: number, item: Anime): number {
     return item.id;
   }
+
+  /**
+   * Handle actions with pagination.
+   * @param event Event received from Pagination component.
+   */
+  public handlePaginationClick(event: PageEvent): void {
```

Comment:
On angular projects we use `on` prefix for event handler names
```suggestion
   * Handle actions with pagination.
   * @param event Event received from Pagination component.
   */
  public onPaginationClick(event: PageEvent): void {
```